acid/features/status/model.py

```diff
@@ -139,7 +139,7 @@ def progress(self):
     def log_url(self):
         if self.result:
             return self.report_url
-        return f'{config["zuul"]["url"].rstrip("/")}/{self.url}'
+        return f'{current_app.config["zuul"]["url"].rstrip("/")}/{self.url}'
```

Comment:
Can we pass this ref URL as argument in model constructor?

Reply:
It's not a simple change - job is created by Job.create(...) called from Buildset.create(...), so we should pass this to Buildset and next to Job. Are you think it make sense?

Reply:
Move host part of URL to template and this should return only URI part.
Template will get zuul url host from controller which has access to current_app.config.